src/mobile/components/layout/bootstrap.coffee

```diff
@@ -48,6 +48,8 @@ module.exports = ->
   if sd.stitch?.renderQueue?
     mountStitch()
 
+  handleNavBarScroll()
```

Comment:
This file is meant to stay lean, but since the nav bar shows up on each page this seemed appropriate.